alternator/stats.cc

```diff
@@ -107,10 +120,14 @@ stats::stats() : api_operations{} {
                     seastar::metrics::description("total number of consumed write units, counted as half units"),{op("Index")})(alternator_label).set_skip_when_empty(),
             seastar::metrics::make_total_operations("filtered_rows_dropped_total", [this] { return cql_stats.filtered_rows_read_total - cql_stats.filtered_rows_matched_total; },
                     seastar::metrics::description("number of rows read and dropped during filtering operations"))(alternator_label).set_skip_when_empty(),
-            seastar::metrics::make_counter("batch_item_count", seastar::metrics::description("The total number of items processed across all batches"),{op("BatchWriteItem")},
+            seastar::metrics::make_counter("batch_item_count_total", seastar::metrics::description("The total number of items processed across all batches"),{op("BatchWriteItem")},
                     api_operations.batch_write_item_batch_total)(alternator_label).set_skip_when_empty(),
-            seastar::metrics::make_counter("batch_item_count", seastar::metrics::description("The total number of items processed across all batches"),{op("BatchGetItem")},
+            seastar::metrics::make_counter("batch_item_count_total", seastar::metrics::description("The total number of items processed across all batches"),{op("BatchGetItem")},
                     api_operations.batch_get_item_batch_total)(alternator_label).set_skip_when_empty(),
+            seastar::metrics::make_histogram("batch_item_count", seastar::metrics::description("Batches size distribution"),{op("BatchGetItem")},
```

Comment:
Please imagine that who ever reads this description doesn't know anything about this metric - especially what "size" means. How about something like
"Histogram of the number of items in a batch request".

Reply:
done